flasher/nina.go

```diff
@@ -86,19 +89,22 @@ func (f *NinaFlasher) FlashFirmware(firmwareFile *paths.Path) error {
 	}
 
 	logrus.Debugf("Checking md5")
-	err = f.md5sum(data)
-	if err != nil {
+	if err := f.md5sum(data); err != nil {
 		logrus.Error(err)
 		return err
 	}
-	logrus.Debugf("Flashed all the things")
-	return nil
+	logrus.Infof("Flashed all the things")
+	flasherOut.Write([]byte("Flashed all the things"))
+	flasherOut.Write([]byte(fmt.Sprintln()))
+	return err //should be nil
 }
 
-func (f *NinaFlasher) FlashCertificates(certificatePaths *paths.PathList, URLs []string) error {
+func (f *NinaFlasher) FlashCertificates(certificatePaths *paths.PathList, URLs []string, flasherOut io.Writer) error {
 	var certificatesData []byte
 	for _, certPath := range *certificatePaths {
 		logrus.Infof("Converting and flashing certificate %s", certPath)
+		flasherOut.Write([]byte(fmt.Sprintf("Converting and flashing certificate %s", certPath)))
+		flasherOut.Write([]byte(fmt.Sprintln()))
```

Comment:
```suggestion
		flasherOut.Write([]byte(fmt.Sprintf("Converting and flashing certificate %s\n", certPath)))
```

same for the other times you do this. It's fine adding a new line at the end of the string.